### PR TITLE
Adds a completion rate to the `data_headings` of performance reports.

### PR DESCRIPTION
**Pull request by pwolfert**

### CODE REVIEW

**Comment by nathanstitt** [comment]
@pwolfert I think you can probably fix the specs by adding `completion_rate:` to the spec's `data_headings` and `data` sections https://github.com/openstax/tutor-server/blob/master/spec/controllers/api/v1/performance_reports_controller_spec.rb#L74-L88

That's what travis is trying to say but it's hard to read.  You can re-run the failing spec locally by running `bundle exec spec spec/controllers/api/v1/performance_reports_controller_spec.rb` from the tutor-server directory

**Comment by Dantemss** [comment]
Yeah ignore the other differences, they aren't real... just add the `completion_rate` to the specs

File: app/subsystems/tasks/performance_report_methods.rb [hunk 1]
```diff
@@ -2,13 +2,20 @@ module Tasks
   module PerformanceReportMethods
     protected
 
+    def completion_fraction(tasks)
+      completed_count = tasks.reduce(0) do |count, task|
+        count + (task.status == 'completed' ? 1 : 0)
+      end
```
Comment (jpslav) [inline app/subsystems/tasks/performance_report_methods.rb]: Is `task` a real `Tasks::Models::Task`? If so, replace with `completed_count = tasks.count(&:completed?)`

**Comment by jpslav** [comment]
Looks good @pwolfert -- only one comment

**Pull request merged**